### ISSUE TITLE
java.lang.Exception: Could not find inject method in exploit class

### ISSUE THREAD

**Issue by kropesiusiak** [VoxelHax/OpenBukloit#21]
C:\Users\majak\Downloads>java -jar OpenBukloit-1.0.8.jar -e C:\Users\majak\OneDrive\Pulpit\Vault\src\main\java\com\vault\vault\Vault.java
Compiling "C:\Users\majak\OneDrive\Pulpit\Vault\src\main\java\com\vault\vault\Vault.java"
External exploit class compiled and loaded: "com.vault.vault.Vault" (class version: 52.0 / Java 8)
java.lang.Exception: Could not find inject method in exploit class any fix or explanation?

**Comment by MsMaciek123** [comment]
Would you mind sharing the exploit code? (possibly without the functions' body, only signatures)

**Comment by wendellmeset** [comment]
Yes, please share the code of the exploit here in order to allow us to debug your error, however based on the error message, I assume your exploit code does not have an inject method.

```java
    public static void inject(JavaPlugin args) {
        System.out.println("Hello, world!");
    }
   ```
